M2/Macaulay2/tests/normal/basis3.m2

```diff
@@ -110,3 +131,16 @@ assert(basis(2, R) == matrix"a2,b")
 -- FIXME: assert(basis(2, R, Truncate => true) == matrix "b,a2,c")
 assert(basis(2, R) == matrix"a2,b")
 
+-- tests for basis of ring maps
```

Comment:
I don't quite understand this example.  What should be fixed?

Reply:
Sorry, which test exactly are you referring to? Maybe your comment is highlighting the wrong block?

ps: you can actually comment on multiple lines at once: click on a line number, say 139, then hold shift and click on another line number, say 142, then crucially, click on the blue + icon next to the _last line_, i.e. 142. It'll say something like:
> Commenting on lines +139 to +142